client/src/pages/Register/Register.tsx

```diff
@@ -0,0 +1,82 @@
+import styles from './Register.module.scss';
+import { GeneralInfo, SignUp, CreateAccount } from '@/components';
+import { Route, Routes, useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+
+export default function Register(): JSX.Element {
+  const navigate = useNavigate();
+  const [formData, setFormData] = useState({
+    firstName: '',
+    lastName: '',
+    userName: '',
+    email: '',
+    phone: '',
+    DoB: '',
+    address: '',
+    city: '',
+    state: '',
+    zip: '',
+    password: '',
+    confirmPassword: '',
+  });
+
+  const updateFormData = (field: string, value: string) => {
+    setFormData((prevData) => ({
+      ...prevData,
+      [field]: value,
+    }));
+  };
+
+  const handleSubmitGeneralInfo = () => {
+    navigate('/register/sign-up');
```

Comment:
We talked about this, but I'm just gonna put this here for documentation.
This series of handles are routing to a new page each time, we don't want to do this for various reasons.
Instead you should keep track of which step your on and change which component is viewed based on that.
And MUI should have stuff to help with that, specifically a "Stepper"

Reply:
@Linkaloo Do you think I can make those changes into a separate PR? I believe there are a couple issues which would implement changes across the files here, and I want to get this merged before they start working.

Reply:
Yea that also works